paper/ref.bib

```diff
@@ -1,8 +1,9 @@
-@inproceedings{legat2017sos,
-  author    = {Legat, Beno{\^\i}t and Coey, Chris and Deits, Robin and Huchette, Joey and Perry, Amelia},
-  title     = {{Sum-of-squares optimization in Julia}},
-  booktitle = {The First Annual JuMP-dev Workshop},
-  year      = {2017},
+@Conference{weisser2019polynomial,
+  author    = {Weisser, Tillmann and Legat, Beno{\^\i}t and Coey, Chris and Kapelevich, Lea and Vielma, Juan Pablo},
+  title     = {Polynomial and Moment Optimization in Julia and JuMP},
+  booktitle = {JuliaCon},
+  year      = {2019},
+  url       = {https://pretalx.com/juliacon2019/talk/QZBKAU/},
```

Comment:
I am not sure I like this change. This is just a presentation, not a paper. I believe there is no paper available.

Reply:
I agree, but I think this is what benoît recommended on the review

Reply:
Recommendations are not requirements... :)
Since you agree, let's undo it?

Reply:
Or maybe we just cite both?

Reply:
maybe we can have both, since it's also what's in SumOfSquares.jl [citation.bib](https://github.com/jump-dev/SumOfSquares.jl/blob/master/CITATION.bib)

Reply:
wait a second, also the previous 2017 reference is a presentation from JuMP-dev, so maybe we can simply keep the new one?